.github/workflows/tests.yaml

```diff
@@ -32,7 +32,8 @@ jobs:
       # Install all dependencies at the root
       - name: Install dependencies (pnpm)
         run: pnpm install
-
+      - name: Check formatting
+        run: pnpm format:check
```

Comment:
wouldn't formatting check be best via a git hooks management tool like [husky](https://typicode.github.io/husky/)?
Given `biome check` doesn't fix the issues detected, wouldn't the detection be best on commit while still in a code editor so that issues can be fixed right then than in ci

Reply:
I'd rather we do that check on CI for a few reasons:
1. That's the practice we follow in the firebase-js-sdk repo
2. If husky isn't installed, then the pre-commit hook doesn't get called and so the check is useless.
3. We also check this in the flutterfire repo: https://github.com/firebase/flutterfire/actions/runs/13930633113/job/38986631698

Reply:
Sounds great. We can keep the check on ci. thanks for the references. Definitely going to check them out.